<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Jogadores</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>160</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="card1_button">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="card2_button">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="card3_button">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="card4_button">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="card5_button">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>543</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Tier1">
    <item>
     <widget class="QPushButton" name="stage1">
      <property name="text">
       <string>Animações</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage2">
      <property name="text">
       <string>TV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage3">
      <property name="text">
       <string>Games</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage4">
      <property name="text">
       <string>Anime e Mangá</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage5">
      <property name="text">
       <string>Livros</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>541</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Tier2">
    <item>
     <widget class="QPushButton" name="stage11">
      <property name="text">
       <string>Anime e Mangá</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage10">
      <property name="text">
       <string>Livros</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage9">
      <property name="text">
       <string>Animações</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage8">
      <property name="text">
       <string>TV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage7">
      <property name="text">
       <string>Games</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>541</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Tier3">
    <item>
     <widget class="QPushButton" name="stage13">
      <property name="text">
       <string>Livros</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage14">
      <property name="text">
       <string>Games</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage15">
      <property name="text">
       <string>Anime e Mangá</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage16">
      <property name="text">
       <string>Animações</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stage17">
      <property name="text">
       <string>Filmes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="questionTitleEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stage6">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>150</y>
     <width>99</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Música</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stage12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>99</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quadrinhos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>680</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardsLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cartas Disponíveis</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>521</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="optionA">
        <property name="text">
         <string>Alternativa A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optionB">
        <property name="text">
         <string>Alternativa B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="optionC">
        <property name="text">
         <string>Alternativa C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optionD">
        <property name="text">
         <string>Alternativa D</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>176</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="player1_name">
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nome do jogador</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="player1_color_label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>330</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FIM</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardsLabel_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fases</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>531</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pergunta:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Categoria:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="questionCategoryLabel">
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Nome da categoria</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
